addons/website_blog/views/website_blog_templates.xml

```diff
@@ -260,13 +252,13 @@ list of filtered posts (by date or tag).
 ============================================================================ -->
 <template id="blog_post_content" name="Blog post content">
     <t t-if="opt_blog_post_breadcrumb and not opt_blog_post_regular_cover" t-call="website_blog.post_breadcrumbs">
-        <t t-set="additionnal_classes" t-value="'mb-3 bg-transparent'"></t>
+        <t t-set="additionnal_classes" t-value="'mb-3 bg-transparent o_container_small pt-4'"></t>
     </t>
     <div t-field="blog_post.content"
         data-editor-message="WRITE HERE OR DRAG BUILDING BLOCKS"
-        t-attf-class="o_wblog_post_content_field #{'js_tweet' if opt_blog_post_select_to_tweet else ''} #{'js_comment' if opt_blog_post_select_to_comment else ''} #{'o_wblog_read_text' if opt_blog_post_readable else ''}"/>
+        t-attf-class="o_wblog_post_content_field #{'js_tweet' if opt_blog_post_select_to_tweet else ''} #{'js_comment' if opt_blog_post_select_to_comment else ''} o_wblog_read_text"/>
 
-    <div t-if="len(blogs) > 1 or len(blog_post.tag_ids) > 0" class="css_editable_mode_hidden text-muted">
+    <div t-if="len(blogs) > 1 or len(blog_post.tag_ids) > 0" class="css_editable_mode_hidden text-muted o_container_small">
```

Comment:
I am scared to see the o_container_small class used in your layout template... should not it be only used inside snippets (blog content) ? What are those ones here ?

Reply:
I didn't want to create another class with the same rules of `o_container_small`
<img width="1679" alt="Capture d’écran 2021-06-30 à 10 50 55" src="https://user-images.githubusercontent.com/18035234/123932066-5e450900-d991-11eb-8944-435dddf6d9f6.png">